docs/sources/administration/plugin-management/index.md

```diff
@@ -228,6 +228,57 @@ WARN[06-01|16:45:59] Running an unsigned plugin   pluginID=<plugin id>
 If you're developing a plugin, then you can enable development mode to allow all unsigned plugins.
 {{% /admonition %}}
 
+## Plugin Frontend Sandbox
+
+The Plugin Frontend Sandbox is a security feature that isolates plugin frontend code from the main Grafana application.
+When enabled, plugins run in a separate JavaScript context, which provides several security benefits:
+
+- Prevents plugins from modifying parts of the Grafana interface outside their designated areas
+- Stops plugins from interfering with other plugins functionality
+- Protects core Grafana features from being altered by plugins
+- Prevents plugins from modifying global browser objects and behaviors
+
+### Enable Frontend Sandbox
+
+The Frontend Sandbox feature is currently behind the `pluginsFrontendSandbox` feature flag. To enable it, you'll need to:
+
+1. Enable the feature flag in your Grafana configuration. For more information about enabling feature flags, refer to [Configure feature toggles](/setup-grafana/configure-grafana/feature-toggles/).
+
+2. For self-hosted Grafana installations, add the plugin IDs you want to sandbox in the `security` section using the `enable_frontend_sandbox_for_plugins` configuration option.
+
+For Grafana Cloud users, you can simply use the toggle switch in the plugin catalog page to enable or disable the sandbox for each plugin. By default, the sandbox is disabled for all plugins.
+
+{{% admonition type="note" %}}
+Enabling the Frontend Sandbox might impact the performance of certain plugins. Only disable the sandbox if you fully trust the plugin and understand the security implications.
+{{% /admonition %}}
+
+### Compatibility
+
+The Frontend Sandbox is compatible with all types of plugins including app plugins, panel plugins, and data source plugins.
+
+The Frontend Sandbox is not compatible with:
+
+- Angular-based plugins
+- Plugins developed and signed by Grafana Labs
```

Comment:
WDYT about this suggestion? I found it a bit weird to say that Grafana plugins are not compatible, while may be, it's just a opinionated decission.

```suggestion
The Frontend Sandbox is compatible with all types of plugins including app plugins, panel plugins, and data source plugins. Angular-based plugins are not supported. Plugins developed and signed by Grafana Labs are excluded and cannot be sandboxed.
```

Reply:
I like it!! added